cmd/livepeer_cli/wizard.go

```diff
@@ -161,16 +162,16 @@ func (w *wizard) readDefaultInt(def int) int {
 	return def
 }
 
-func (w *wizard) readBigInt() *big.Int {
+func (w *wizard) readBigInt(prompt string) *big.Int {
+	if prompt == "" {
```

Comment:
Do we need this check? I don't think we ever use `readBigInt("")`.

Reply:
I think you're right about this. Probably will never call it with empty string.